src/mo_types/type.mli

```diff
@@ -78,6 +78,7 @@ end
 
 val is_shared_sort : 'a shared -> bool
 
+
```

Comment:
was this intentional?

Reply:
The space? Probably not.

Reply:
Actually it was - Andreas left two space before sections. Adding some more ;->  in #3723